Source/Core/MortonOrder.js

```diff
@@ -0,0 +1,85 @@
+import Check from "./Check.js";
+import defined from "./defined.js";
+import DeveloperError from "./DeveloperError.js";
+
+/**
+ * Morton Order (aka Z-Order Curve) helper functions.
+ * @see {@link https://en.wikipedia.org/wiki/Z-order_curve}
+ * @see {@link http://graphics.stanford.edu/~seander/bithacks.html#InterleaveBMN}
+ *
+ * @namespace MortonOrder
+ */
+var MortonOrder = {};
+
+/**
+ * @private
+ * @param {Number} v
+ */
+function insertSpacingBetweenBits(v) {
```

Comment:
Can these functions be used in the future for interleaving 3D coordinates, or will they need to be modified?